src/components/PresetEditor/PresetEditor.css

```diff
@@ -2,7 +2,6 @@
   display: flex;
   justify-content: space-evenly;
   text-align: center;
-  height: 100%;
```

Comment:
I dont _think_ this has any side effects

Reply:
we'll find out woooo